src/k8s-extension/azext_k8s_extension/_client_factory.py

```diff
@@ -7,15 +7,31 @@
 from azure.cli.core.profiles import ResourceType
 
 
-def cf_k8s_extension(cli_ctx, *_):
+def cf_k8s_extension(cli_ctx, **kwargs):
     from .vendored_sdks import SourceControlConfigurationClient
-    return get_mgmt_service_client(cli_ctx, SourceControlConfigurationClient)
+    return get_mgmt_service_client(cli_ctx, SourceControlConfigurationClient, **kwargs)
 
 
 def cf_k8s_extension_operation(cli_ctx, _):
     return cf_k8s_extension(cli_ctx).extensions
 
 
+def cf_k8s_cluster_extension_types_operation(cli_ctx, _):
+    return cf_k8s_extension(cli_ctx, api_version='2022-01-15-preview').cluster_extension_types
+
+
+def cf_k8s_cluster_extension_type_operation(cli_ctx, _):
+    return cf_k8s_extension(cli_ctx, api_version='2022-01-15-preview').cluster_extension_type
+
+
+def cf_k8s_location_extension_types_operation(cli_ctx, _):
+    return cf_k8s_extension(cli_ctx, api_version='2022-01-15-preview').location_extension_types
+
+
+def cf_k8s_extension_type_versions_operation(cli_ctx, _):
+    return cf_k8s_extension(cli_ctx, api_version='2022-01-15-preview').extension_type_versions
```

Comment:
If the `api_version` used by these operations need to be consistent, could we abstract `2022-01-15-preview` into a reusable constant